assets/src/scripts/monitoring-location/components/hydrograph/parameters.js

```diff
@@ -17,109 +15,218 @@ import {getAvailableParameters} from './selectors/parameter-data';
-        let selection = select(this);
+/*
+* Helper function that creates the top row of each parameter selection. This row is hidden except on narrow screens
+* and contains the period of record that appears above the parameter description.
+* @param {Object} Element - The target element to append the row
+* @param {Object} parameter - Contains details about the current parameter code
+* @return {Object} The HTML for the grid row
+* */
+const drawTopPeriodOfRecordRow = function(element, parameter) {
+    const gridRowInnerTopPeriodOfRecord = element.append('div')
+        .attr('class', 'grid-row grid-row-inner grid-row-period-of-record');
+    gridRowInnerTopPeriodOfRecord.append('div')
+        .attr('class', 'grid-col-10 grid-offset-1')
+        .text(`${parameter.periodOfRecord.begin_date} to ${parameter.periodOfRecord.end_date}`);
+    const TopPeriodOfRecordRowExpansionControlDiv = gridRowInnerTopPeriodOfRecord.append('div')
+        .attr('class', 'grid-col-1 open-close-top-period-of-record');
 
-        let waterAlertCell;
-        if (d.waterAlert.hasWaterAlert) {
-            waterAlertCell = selection.append('a')
-                .attr('href', `${config.WATERALERT_SUBSCRIPTION}/?site_no=${siteno}&parm=${d.waterAlert.subscriptionParameterCode}`);
-        } else {
-            waterAlertCell = selection;
-        }
-        waterAlertCell.attr('class', 'water-alert-cell usa-tooltip')
+    drawRowExpansionControl(TopPeriodOfRecordRowExpansionControlDiv, parameter, 'mobile');
+};
+
+/*
+* Helper function that draws the row containing the radio button and parameter description.
+* @param {Object} Store - The application Redux state
+* @param {Object} Element - The target element to append the row
+* @param {Object} parameter - Contains details about the current parameter code
+*/
+const drawRadioButtonRow = function(store, element, parameter) {
+    const gridRowInnerWithRadioButton = element.append('div')
+        .attr('class', 'grid-row grid-row-inner');
+    const radioButtonDiv = gridRowInnerWithRadioButton.append('div')
+        .attr('class', 'grid-col-1 radio-button__param-select')
+        .append('div')
+        .attr('class', 'usa-radio');
+    radioButtonDiv.append('input')
+        .attr('class', 'usa-radio__input')
+        .attr('id', `radio-${parameter.parameterCode}`)
+        .attr('type', 'radio')
+        .attr('name', 'parameter-selection')
+        .attr('value', `${parameter.parameterCode}`)
+        .call(link(store, (inputElem, selectedParameterCode) => {
+            inputElem.property('checked', parameter.parameterCode === selectedParameterCode ? true : null);
+        }, getSelectedParameterCode));
+    radioButtonDiv.append('label')
+        .attr('class', 'usa-radio__label')
+        .attr('for', `radio-${parameter.parameterCode}`);
+    gridRowInnerWithRadioButton.append('div')
+        .attr('class', 'grid-col-7 description__param-select')
+        .append('div')
```

Comment:
This div seems unnecessary. Can't we add the parameter description directly to the .description__param-select div.

Reply:
how'd that sneak in there? Good catch.